tests/test_algorithm.py

```diff
@@ -1595,13 +1625,15 @@ def tearDownClass(cls):
         cls.tempdir.cleanup()
 
     def test_auto_close_future(self):
-        self.algo = TestAlgorithm(sid=1, amount=1, order_count=1,
-                                  instant_fill=True, commission=PerShare(0),
```

Comment:
sorry, this is a test I'm still working on.  didn't mean to commit these changes, but I'll just fix up the test afterwards, since you need this PR.